packages/orbit-components/src/Dialog/Dialog.stories.tsx

```diff
@@ -1,81 +1,124 @@
 import * as React from "react";
 import { action } from "@storybook/addon-actions";
+import type { Meta, StoryObj } from "@storybook/react/";
 
 import Button from "../Button";
 import ButtonLink from "../ButtonLink";
 import Illustration from "../Illustration";
 import { NAMES } from "../Illustration/consts.mts";
+import type { Name } from "../Illustration/types";
 
 import Dialog from ".";
 
-export default {
+const meta: Meta<typeof Dialog> = {
   title: "Dialog",
-};
-
-export const Default = ({ title, description, dataTest }) => {
-  return (
-    <Dialog
-      title={title}
-      description={description}
-      dataTest={dataTest}
-      primaryAction={<Button type="critical">Log out</Button>}
-      renderInPortal={false}
-    />
-  );
-};
+  component: Dialog,
 
-Default.story = {
   parameters: {
     info: "Visit Orbit.Kiwi for more detailed guidelines.",
   },
-};
 
-Default.args = {
-  title: "Log out",
-  description: "Are you sure you want to log out now?",
-  dataTest: "test",
-};
+  args: {
+    title: "Log out",
+    description: "Are you sure you want to log out now?",
+    illustration: "NoNotification",
+    maxWidth: NaN,
+    renderInPortal: false,
+    lockScrolling: false,
+    onClose: action("onClose"),
+  },
 
-export const Playground = ({ title, maxWidth, description, illustration, dataTest }) => {
-  return (
-    <Dialog
-      title={title}
-      maxWidth={maxWidth}
-      description={description}
-      illustration={<Illustration name={illustration} size="small" />}
-      dataTest={dataTest}
-      onClose={action("onClose")}
-      primaryAction={<Button type="critical">Log out</Button>}
-      secondaryAction={<ButtonLink type="secondary">Cancel</ButtonLink>}
-      renderInPortal={false}
-    />
-  );
+  argTypes: {
+    illustration: {
+      options: NAMES,
+      control: {
+        type: "select",
+      },
+    },
+    maxWidth: {
+      control: {
+        type: "number",
+      },
+    },
+  },
 };
 
-Playground.story = {
+export default meta;
+type Story = StoryObj<typeof Dialog>;
+
+function useDialog() {
+  const [open, toggle] = React.useReducer((_, next) => next, true);
```

Comment:
Could you please explain the reason behind using `useReducer` here?

Reply:
I just used the same function as it was used in [Modal.stories.tsx](https://github.com/kiwicom/orbit/blob/5c87463e700a0bd7879259cc478197d4293fe3a0/packages/orbit-components/src/Modal/Modal.stories.tsx#L35) file, but I think using `useState` should be enough for this case. 🤔  WDYT?

Reply:
yes, that's true. going to change it ASAP. :)

Reply:
Changed. I've also changed:
```
<div style={{ display: open ? "block" : "none" }}>{children}</div>
```

to

```
{open && <div>{children}</div>
```

to make the code little bit simpler.